features-json/webgpu.json

```
@@ -527,7 +527,10 @@
       "73":"n"
     },
     "and_chr":{
-      "117":"n"
+      "116":"n",
+      "117":"n",
+      "118":"n d #3",
+      "119":"n d #3"
```

```suggestion
      "119":"n d #3",
      "120":"n d #3",
      "121":"y"
```
It's now available!
Source: https://developer.chrome.com/blog/new-in-webgpu-121